hoot-core-test/src/test/cpp/hoot/core/elements/TagsTest.cpp

```diff
@@ -84,6 +85,34 @@ class TagsTest : public CppUnit::TestFixture
     HOOT_STR_EQUALS("[3]{foo;bar, baz;qux, b}", t.getList("a"));
     HOOT_STR_EQUALS("foo;;bar;baz;;qux;b", t["a"]);
   }
+
+  void unitsTest()
+  {
+    Tags t1, t2, t3, t4, t5, t6, t7, t8;
+    t1.setList("length", QStringList() << "60 mi");
```

Comment:
No need to change it, but rather than using setList you can use set. E.g.

t1.set("length", "60 mi");

or use the [] overload:

t1["length"] = "60 mi";